### ISSUE TITLE
Cannot use insider program even after using RestoreAll

### ISSUE THREAD

**Issue by v0idst4r** [Disassembler0/Win10-Initial-Setup-Script#67]
Windows 10 insider program shows "Your diagnostic and usage data settings require attention to get Insider Preview builds." and a "Fix me" button that was greyed out before I did RestoreAll, now I can press it but it does nothing. I've tried relinking my insider account as well. Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\DataCollection\AllowTelemetry is set to 3. The diagnostic data option in settings is set to full but greyed out. "*Some settings are hidden or managed by your orginization." I've also tried setting the Allow Telemetry group policy to Enabled and 3 (it was not configured) but had no effect.

**Comment by Disassembler0** [comment]
> The diagnostic data option in settings is set to full but greyed out. "*Some settings are hidden or managed by your orginization."

Yeah, this is a regression in `EnableTelemetry` which I will fix in next release. For now, just remove the `HKLM\SOFTWARE\Policies\Microsoft\Windows\DataCollection\AllowTelemetry` key altogether as shouldn't be there in the default configuration.

Apart from `EnableTelemetry`, you need also `EnableFeedback` and `EnableAdvertisingID` (resp. just the value of `TailoredExperiencesWithDiagnosticDataEnabled` key which I will move from `EnableAdvertisingID` to `EnableTelemetry` in next release), but since you've already run RestoreAll (which isn't really recommended as it's mainly for testing purposes), you should already have them applied.
